test/regression/solvers/delta_eddington.cpp

```diff
@@ -0,0 +1,206 @@
+// Copyright (C) 2023-2024 National Center for Atmospheric Research
+// SPDX-License-Identifier: Apache-2.0
+//
+// Tests for tuvx::DeltaEddington.
+#include "delta_eddington.hpp"
+#include <iostream>
+
+#define ASSERT(x) if (!(x)) { \
```

Comment:
would it be possible to use gtest asserts here even though gtest isn't running this test?

Reply:
Do the gtest asserts work in Fortran? I just assumed they didn't.